packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -208,6 +211,80 @@ class SelectDefaultSr extends Component {
   }
 }
 
+@connectStore(
+  () => ({
+    haSrs: createGetObjectsOfType('SR').pick((_, { srs }) => srs),
+  }),
+  { withRef: true }
+)
```

Comment:
If you can pass IDs to `SelectSr`, you don't need to resolve the SR objects at all anymore, right? Since you're only handling IDs, now.